backend/utils/config.py

```diff
@@ -1,12 +1,13 @@
 import os
 from dotenv import load_dotenv
 
+
 class Config(object):
     def __init__(self):
         self.frontend_url = None
-        self.mistral_url = None
-        self.mistral_key = None
-        self.mistral_model = None
+        self.mistralruf_url = None
+        self.mistralruf_key = None
+        self.mistralruf_model = None
```

Comment:
why are we adding `ruf` to the property names here?

Reply:
Yeah I will change this, think it might have done this on one of the auto-fixes, but these have now been removed as well